src/app/components/Transaction/index.tsx

```diff
@@ -160,6 +161,21 @@ export function Transaction(props: TransactionProps) {
         ),
       },
     },
+    [transactionTypes.TransactionType.ConsensusDeposit]: {
+      [TransactionSide.Received]: unrecognizedTransaction,
+      [TransactionSide.Sent]: {
+        designation: t('common.to', 'To'),
+        icon: () => <Inherit />,
+        header: () => (
+          <Trans
+            i18nKey="account.transaction.consensusDeposit.sent"
+            t={t}
+            components={[transaction.runtimeName]}
+            defaults={`<0></0> deposit`}
```

Comment:
This would be safer as
```suggestion
            values={{ runtimeName: transaction.runtimeName }}
            defaults={`{{runtimeName}} deposit`}
```

(components technically allows JSX)

Reply:
Should be resolved.